discovery_engine_core/core/src/storage/migrations/20220823100617_source_reaction.sql

```diff
@@ -17,6 +17,4 @@ CREATE TABLE IF NOT EXISTS SourceReaction (
     weight INTEGER NOT NULL,
     -- format should be RFC3339/ISO8601 & sqlite compliant
     lastUpdated TEXT NOT NULL,
```

Comment:
:wrench: sql sadly doesn't allow trailing commas

```suggestion
    lastUpdated TEXT NOT NULL
```

Reply:
aha thanks! was wondering where I went wrong.
BTW also added `test_source_reaction` to the sqlite tests